app/controllers/holds_controller.rb

```diff
@@ -132,8 +125,4 @@ def deny_create
     def barcodes
       [params['barcodes']].flatten.compact
     end
-
-    def item_details
-      { holdRecordList: true }
-    end
```

Comment:
no need for this anymore? what about the one in checkouts_controller?

Reply:
🤔 do we? I wasn't sure tbh, but page loads were fast to the spinner. I'll double check. This happened because a straight reverse in git is what I did of both of the previous PRs.

Reply:
I am not sure either, we are passing it in the job as well. thanks for checking, I am curious.

Reply:
no we can't have this, and I removed the one that was still there for `checkouts_controller.rb`. The reason is that on load of `/checkouts` (or `/holds`), the first thing that happens is `authenticate_user!` - this will in turn look up the current user and make a `Patron` out of it. To make a patron it uses `SymphonyClient.patron_info`. That method gets passed `item_details` on the controller being requested. This method will dictate what parameters are sent to the web service. The default was made to be _only_ `alternateId` so it could be super speedy on first load. If you pass in `{circRecordList: true }` it'll want to get all the checkouts too. 

Checkouts, in this sidekiq approach, are pulled in by a job instead, that job directly sets the requirement for what is needed. (i.e.,`{ circRecordList: true }`.

Reply:
yes, makes sense 👍  thanks!